policyengine_canada/variables/gov/provinces/yt/tax/income/benefits/yukon_child_benefit/yt_child_benefit_post_reduction.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_canada.model_api import *
+
+
+class yt_child_benefit_post_reduction(Variable):
+    value_type = float
+    entity = Household
+    label = "Yukon child benefit post reduction"
```

Comment:
Let's label final variables with their normal name, since it's what shows in the app
```suggestion
    label = "Yukon child benefit"
```